src/main/resources/assets/dndecor/lang/ru_ru.json

```
@@ -422,36 +426,36 @@
-  "block.dndecor.yellow_velvet_block": "Жёлтый бархат",
+  "block.dndecor.yellow_velvet_block": "Жёлтый блок обивки",
   "block.dndecor.zinc_bricks": "Цинковые кирпичи",
   "block.dndecor.zinc_checker_tiles": "Цинковая шахматная плитка",
   "block.dndecor.zinc_cross_bolt": "Цинковый крестошлицевой болт",
-  "block.dndecor.zinc_dash_bolt": "Цинковый линейчатый болт",
-  "block.dndecor.zinc_dot_bolt": "Цинковый точечный болт",
-  "block.dndecor.zinc_flat_bolt": "Цинковый плоскоголовый болт",
-  "block.dndecor.zinc_floor": "Цинковый пол",
+  "block.dndecor.zinc_dash_bolt": "Цинковый плоскошлицевой болт",
+  "block.dndecor.zinc_dot_bolt": "Цинковый точечношлицевой болт",
+  "block.dndecor.zinc_flat_bolt": "Цинковый бесшлицевой болт",
+  "block.dndecor.zinc_floor": "Цинковый фабричный пол",
   "block.dndecor.zinc_frontlight": "Цинковый фасадный светильник",
-  "block.dndecor.zinc_frontlight.tooltip.behaviour1": "Фасадный светильник загорается.",
-  "block.dndecor.zinc_frontlight.tooltip.behaviour2": "Фасадный светильник поворачивается на 45 градусов.",
-  "block.dndecor.zinc_frontlight.tooltip.behaviour3": "Тип фасадного светильника изменяется.",
+  "block.dndecor.zinc_frontlight.tooltip.behaviour1": "Загорается!",
+  "block.dndecor.zinc_frontlight.tooltip.behaviour2": "Поворачивает светильник на 45 градусов.",
+  "block.dndecor.zinc_frontlight.tooltip.behaviour3": "Изменяет внешний вид лицевой стороны светильника.",
   "block.dndecor.zinc_frontlight.tooltip.condition1": "При подаче редстоун-сигнала сзади",
-  "block.dndecor.zinc_frontlight.tooltip.condition2": "ПКМ гаечным глючом спереди",
-  "block.dndecor.zinc_frontlight.tooltip.condition3": "ПКМ гаечным глючом сбоку",
-  "block.dndecor.zinc_frontlight.tooltip.summary": "Легконастраиваемый декоративный блок.",
-  "item.design_decor.storage_container": "Контейнер для хранения",
-  "item.dndecor.belt_connector": "Механический ремень (полный)",
-  "itemGroup.dndecor.base": "Design n' Decor",
-  "tag.item.dndecor.dark_metal_decor": "Декор из тёмного металла",
-  "tag.item.dndecor.dyed_stone_metal_decor": "Декор из окрашенного металлизированного камня",
-  "tag.item.dndecor.dyed_velvet_block": "Окрашенный бархат",
+  "block.dndecor.zinc_frontlight.tooltip.condition2": "ПКМ по лицевой стороне блока гаечным глючом",
+  "block.dndecor.zinc_frontlight.tooltip.condition3": "ПКМ по боковой стороне блока гаечным глючом",
+  "block.dndecor.zinc_frontlight.tooltip.summary": "Декоративный источник света с настраиваемым внешним видом.",
+  "item.design_decor.storage_container": "Хранилище-контейнер",
+  "item.dndecor.belt_connector": "Механическая конвейерная лента",
+  "itemGroup.dndecor.base": "Create: Design n' Decor",
+  "tag.item.dndecor.dark_metal_decor": "Декоративные блоки из тёмного металла",
+  "tag.item.dndecor.dyed_stone_metal_decor": "Декоративные блоки из окрашенного металлизированного камня",
+  "tag.item.dndecor.dyed_velvet_block": "Окрашенный блок обивки",
```

В тегах принято описывать множественным числом по возможности
```suggestion
  "tag.item.dndecor.dyed_velvet_block": "Окрашенные блоки обивки",
```

---

> В тегах принято описывать множественным числом по возможности

100% понимания, это просто массовая замена подвела :P